### PR TITLE
Added basic implementation for textDocument/definition action

### PR DESCRIPTION
**Pull request by svermeulen**

### CODE REVIEW

File: src/handlers.tl [hunk 1]
```diff
@@ -76,6 +77,47 @@ handlers["textDocument/didSave"] = function(params: Params)
...
+   local info <const> = doc:type_information_at(pos)
+
+   if not info or info.file == nil then
+      rpc.respond(id, nil)
+      return
+   end
+
+   util.log("Found type info: ", info)
+
+   local file_uri:uri.ParsedUri
```
Comment (euclidianAce) [inline src/handlers.tl:100]: small style typo, should be a space between `file_uri:` and `uri.ParsedUri`

File: src/path_util.tl [hunk 2]
```diff
@@ -0,0 +1,31 @@
+
```
Comment (euclidianAce) [inline src/path_util.tl:1]: this project depends on [cyan](github.com/teal-language/cyan), which has a decent [path module](https://teal-language.github.io/cyan/index.html#cyan.fs.path) that can do these operations

**Review by euclidianAce — CHANGES_REQUESTED** [review]
Other than the couple of comments I left, looks good! :+1:

**Comment by lewis6991** [comment]
Is there anything blocking this now?

**Review by euclidianAce — APPROVED** [review]

**Comment by euclidianAce** [comment]
Sorry for not getting to this quicker! Been real busy as of late. Thanks!

**Pull request merged**